### PR TITLE
Feature/projectsui

### PR DESCRIPTION
**Pull request by caberoDaniel**

### CODE REVIEW

File: src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                          final ProjectSettings projectSettings) {
+        this.initHomePage = homePage;
+        this.initDastBoard = dashBoard;
+        this.projectSettings = projectSettings;
+    }
+
+    /**
+     * @param user to login to the pivotaltracker.
+     */
+    @Given("^I login to pivotalTracker as \"([^\"]*)\"$")
```
Comment (pablopzsj) [inline src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java:36]: Use the hook

File: src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    public void iLoginToPivotalAs(final String user) {
+        initDastBoard = SignIn.loginOtherUser();
+    }
+
+    /**
+     * Method to navidate to the dashboard.
+     */
+    @And("^I navigate dashboard$")
```
Comment (pablopzsj) [inline src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java:44]: Use the hook

File: src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+     * @param name2 atribute for update a name1.
+     */
+    @When("^I update \"([^\"]*)\" with \"([^\"]*)\":$")
+    public void iUpdateWith(final String name1, final String name2) {
+        initDastBoard.searchForAProject(name1);
+        initDastBoard.clickOnProjectSettingButton();
+        projectSettings.updateProjectName(name2);
+        projectSettings.setProjectDescription(name2);
+        projectSettings.clickOnSaveProjectButton();
+        projectSettings.clickonGotoHome();
```
Comment (pablopzsj) [inline src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java:72]: Use the hook "goToDashboard"

File: src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+     * @param action  is the method two differents actions
+     * @param name    atribute do action sending.
+     */
+    @When("^I \"([^\"]*)\" the \"([^\"]*)\"$")
+    public void iThe(final String action, final String name) {
+        initDastBoard.searchForAProject(name);
+        initDastBoard.clickOnProjectSettingButton();
+        projectSettings.clickOnArchive();
+        projectSettings.clickOnConfirmArchive();
+        projectSettings.clickonGotoHome();
```
Comment (pablopzsj) [inline src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java:86]: Use the hook "goToDashboard"

File: src/test/resources/features/ui/projects/Project1.feature [hunk 5]
```diff
@@ -0,0 +1,23 @@
+Feature: project delete archive and unarchive
+
+  Background:
+    Given I login to pivotalTracker as "user1"
+    And I request "POST" "/projects" with:
+      | name | project255511234564 |
+    And I stored as [project1]
+
+  @gotodasboard
```
Comment (pablopzsj) [inline src/test/resources/features/ui/projects/Project1.feature:9]: is @goToDashboard

File: src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java (hunk 1 above)
Comment (caberoDaniel) [reply src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/StepDefinition.java:36]: yes, I forgot to remove the methods I no longer use

File: src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/AssertionsStepsProjects.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+   * @param step in the constructor StepDefinition.
+   */
+  public AssertionsStepsProjects(final StepDefinition step) {
+    this.step = step;
+  }
+
+  /**
+   * @param message verify the message is correct.
+   */
+  @Then("^Message should be \"([^\"]*)\"$")
```
Comment (caberoDaniel) [inline src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/ui/projects/AssertionsStepsProjects.java]: I'm improving but I still lack some strategies

**Comment by carledriss** [comment]
Solve conflicts

**Pull request closed without merging**